package.json

```diff
@@ -108,5 +108,8 @@
         "nextVersion": "Unreleased",
         "ignoreCommitters": [],
         "feature": "New Feature"
+    },
+    "dependencies": {
+        "recast-navigation": "^0.39.0"

```

Comment:
Marking this so it wouldn't be merged - please don't add dependencies on the main package.json (only dev-dependencies needed for the monorepo). If it is a dependency for a specific package, add it to the specific package, however, as mentioned offline - it would be good not to have any dependency added to our core library. Instead, handle dependencies like we handle draco/havok/csg2/recast1